lib/cleanup.js

```diff
@@ -4,15 +4,23 @@ const BbPromise = require('bluebird');
 const fse = require('fs-extra');
 
 module.exports = {
-  cleanup() {
-    const webpackOutputPath = this.webpackOutputPath;
+	cleanup() {
+		const webpackOutputPath = this.webpackOutputPath;
+		if (this.serverless.utils.dirExistsSync(webpackOutputPath)) {
+      
+      const cli = this.options.verbose ? this.serverless.cli : null;
```

Comment:
Rather then returning null and having to check everywhere before logging it might be easier to return an object with a void function here. Something like `{ log: () => {} }`. That would let you just call `cli.log` without the check making the subsequent code a little less verbose.